### ISSUE TITLE
<p:dataTable> using scrollable and scrollHeight attributes causes Uncaught TypeError

### ISSUE THREAD

**Issue by Three97** [primefaces/primefaces#419]
In Chrome (v.43.0.2357.65), Firefox (v.28.0 & 38.0.5) and Opera (v.30.0) using PrimeFaces v5.2's DataTable works great when the vertical scrollbar is not enabled. However, when it is enabled, getting "Uncaught TypeError: Cannot read property 'webkit' of undefined". The code fails with the "$.browser" property as it is not defined. Appears to be "scrollable" related, but was not able to isolate to horizontal or vertical.

Using Eclipse Luna SR1 (4.4.1) with OEPE & WebLogic v12.1.3.2 on Windows 7 platform.

**Comment by cagataycivici** [comment]
Is it possible to send a test.xhtml and TestView.java to replicate?

**Comment by Three97** [comment]
Thank you for the reply. I will try to abstract the page data so as not to impose on potentially sensitive data and provide a stand-alone example. Hopefully in the next day or two?

**Comment by cagataycivici** [comment]
Sure, I've found a reference to $.browser, we usually use PrimeFaces.env.browser but just want to make sure we fix your issue.

**Comment by Three97** [comment]
I've seen references to replacing $.browser with PrimeFaces.env.browser in some of the recent commits, so I suspect replacing the former with the latter would fix it, but not 100% sure, of course. I was attempting to find the issue more specifically by grabbing the latest and debugging locally, but have not been successful in getting it to run yet...

**Comment by cagataycivici** [comment]
Ok, I'll just do the same and close this one, it should fix it I think.

**Comment by Three97** [comment]
I'm afraid I may have reported an issue that no longer exists...I pulled version 5.2 using Maven, but according to this post, it appears to have been addressed: http://forum.primefaces.org/viewtopic.php?f=3&t=41656. I apologize for bringing this up if that's the case. Not sure what version is being used in the showcase, but that may explain why the error isn't showing up there (new version with fix), but is in my case (older version without fix). I do hope this didn't cause any undue confusion. This may be a different component using a similar mechanism for identifying the browser, though.
